deployment/constructs/lambda.ts

```diff
@@ -102,7 +103,9 @@ export class Lambda extends Construct {
         method,
         new apigateway.LambdaIntegration(this.lambda),
         {
-          authorizationType: apigateway.AuthorizationType.IAM,
+          authorizationType: !isLocalStack()
+            ? apigateway.AuthorizationType.IAM
+            : undefined,
```

Comment:
```suggestion
          authorizationType: isLocalStack()
            ? undefined
            : apigateway.AuthorizationType.IAM,
```
Just reversing to remove the not
Optional nitpick